package.json

```diff
@@ -34,15 +34,15 @@
     "trim": ">=0.0.3"
   },
   "devDependencies": {
-    "@babel/core": "^7.17.10",
-    "@babel/plugin-proposal-class-properties": "^7.14.5",
-    "@babel/preset-env": "^7.17.10",
-    "@babel/preset-typescript": "^7.14.5",
-    "@changesets/cli": "^2.19.0",
-    "@emotion/babel-plugin": "^11.3.0",
-    "@emotion/react": "11.0.0",
```

Comment:
since this is a peerdep, we should keep this pinned at the minimum version in the peerdep range

Reply:
good spot - have pushed a change